app/models/concerns/tsql_import.rb

```diff
@@ -1,9 +1,11 @@
-###
+# frozen_string_literal: true
+
 # Copyright 2016 - 2025 Green River Data Analysis, LLC
 #
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
+# DEPRECATION WARNING: This module may be deprecated in favor of the activerecord-import gem
```

Comment:
double check me on this deprecation.

Reply:
I don't believe that's 100% true, as we still rely on this for importing to SQL Server (LSA & HIC).  
I agree for any postgres import, we should be using either the activerecord-import gem or the active record native upsert mechanism.